hack/tests/e2e-create-prod.sh

```diff
@@ -1,23 +1,8 @@
 #!/bin/bash -e
 
-cleanup() {
```

Comment:
it doesn't make sense to use the same cleanup() for the production e2e tests for reasons explained in https://github.com/openshift/openshift-azure/pull/1625

Reply:
better this way?